docfiles/style.css

```diff
@@ -186,6 +186,30 @@ svg {
     padding-bottom: 3em;
 }
 
+#accessibleMenu {
+    position: absolute;
+    z-index: 1001;
+    top: -20em;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+}
+
+#accessibleMenu a {
+    position: absolute;
+    width: 100%;
+    height: 4em;
+    line-height: 4em;
+    font-size: 1em;
+    text-align: center;
+    color: white;
+}
+
+#accessibleMenu a:focus,
+#accessibleMenu a:hover {
+    top:20em;
+}
+
```

Comment:
Why don't we use the same css for both the docs and the editor? 
semantic.css is loaded in the docs.